src/options/options_handler.cpp

```diff
@@ -96,8 +97,7 @@ Languages currently supported as arguments to the --output-lang option:
   tptp                           TPTP format
   ast                            internal format (simple syntax trees)
 )FOOBAR" << std::endl;
-    std::exit(1);
-    return Language::LANG_AUTO;
+    throw OptionException("help is not a valid language");
```

Comment:
Why `help`?

Reply:
We need some way to abort the solver here for `--lang help`. I want to avoid actually aborting the binary right here, but instead communicate this to the API via an exception. It may very well not the proper exception, though.

Reply:
I missed that this is the `help` case. `OptionException` looks like the right one to me.